schemas/ispyb/stored_programs/sp_retrieve_screenings_for_sample.sql

```diff
@@ -0,0 +1,26 @@
+DELIMITER ;;
+CREATE OR REPLACE DEFINER= `ispyb_root`@`%` PROCEDURE `retrieve_screenings_for_sample`(p_id int unsigned)
```

Comment:
If the joins are correct, then technically this is something like `retrieve_screenings_for_sub_samples_in_sample`. See comment below.

Reply:
I'm going with you on comment below so this doesn't need changing